neo/app/controllers/Application.scala

```diff
@@ -41,7 +41,7 @@ class Application @Inject()(val wsClient: WSClient,
 
   val manager = system.actorOf(DataStoreManager.props(Migration_20160814.berryMeta, asterixConn, AQLGenerator, config))
 
-  Logger.logger.info("I'm initializing")
+  Logger.info("I'm initializing")
```

Comment:
That would be weird. I indeed saw them in the `logs/application.log`. I think it should be something wrong with our logback.xml settings. You can have a try to see if what is a good setting 🤔